dm/loader/lightning.go

```diff
@@ -78,13 +78,17 @@ type LightningLoader struct {
 // NewLightning creates a new Loader importing data with lightning.
 func NewLightning(cfg *config.SubTaskConfig, cli *clientv3.Client, workerName string) *LightningLoader {
 	lightningCfg := makeGlobalConfig(cfg)
+	logger := log.L()
+	if cfg.FrameworkLogger != nil {
+		logger = log.Logger{Logger: cfg.FrameworkLogger}
+	}
 	loader := &LightningLoader{
 		cfg:                   cfg,
 		cli:                   cli,
 		workerName:            workerName,
 		lightningGlobalConfig: lightningCfg,
 		core:                  lightning.New(lightningCfg),
-		logger:                log.With(zap.String("task", cfg.Name), zap.String("unit", "lightning-load")),
+		logger:                logger.WithFields(zap.String("task", cfg.Name), zap.String("unit", "lightning-load")),
```

Comment:
Seems that zap.Logger has `With` method, 
Why does `dm` add a method called `WithFields`?

Reply:
I also don't know. we can change it later